autoscale/src/main/java/com/sequenceiq/periscope/repository/ClusterPertainRepository.java

```diff
@@ -0,0 +1,20 @@
+package com.sequenceiq.periscope.repository;
+
+import java.util.List;
+
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+
+import com.sequenceiq.cloudbreak.workspace.repository.EntityType;
+import com.sequenceiq.periscope.domain.ClusterPertain;
+
+@EntityType(entityClass = ClusterPertain.class)
+public interface ClusterPertainRepository extends CrudRepository<ClusterPertain, Long> {
+    List<ClusterPertain> findByUserCrn(@Param("userCrn") String userCrn, Pageable page);
+
+    default List<ClusterPertain> findByUserCrn(String userCrn) {
```

Comment:
It much better: `ClusterPertain findByUserCrn(String userCrn)` and should be work

Reply:
agree update existing cluster->clusterpertain mapping to support ManyToOne and api to ClusterPertain findByUserCrn(String userCrn)